projects/demo/src/modules/components/accordion/index.ts

```diff
@@ -16,14 +15,13 @@ import {TuiAccordionExample5} from './examples/5';
 @Component({
     standalone: true,
     imports: [
-        TuiAddonDoc,
+        TuiDemo,
         TuiAccordion,
         TuiAccordionExample1,
```

Comment:
Why do we still use old approach in accordion?

Reply:
I don't know why, need fix it in next PR